draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1025,12 +1076,14 @@ inventory/topology/service change occurs.
 
 It should also be possible to correlate information from IP and
 optical layers (e.g., which port, lambda/OTSi, and direction are used
-by a specific IP service on the WDM equipment).
+by a specific IP service on the DWDM equipment).
```

Comment:
There is no definition of 'WDM equipment' or 'DWDM equipment'

I think we can use the term 'WDM node' from I-D.ietf-ccamp-optical-impairment-topology-yang as in the rest of the document

```suggestion
by a specific IP service on the WDM node).
```